### DISCUSSION TITLE
Dependant select input didn't work

### DISCUSSION THREAD

**Discussion by hellohendri** [filamentphp/filament#1976] [Help] [answered]
I need to get price based on product that user selected but my dependant select input didn't work, here's my code :

![dependant-select](https://user-images.githubusercontent.com/92468957/159659455-3e26492b-1894-4d85-98ae-216dcc7b56e0.png)

Error code :

![image](https://user-images.githubusercontent.com/92468957/159660483-72613e63-352e-4d2b-8430-1bb566b386cc.png)

Am I missing something?

**Comment by danharrin** [comment, marked as answer]
`collable` => `callable` or `Closure`

**Reply by hellohendri** [reply]
Oops typo 😅 Thanks!
